background.js

```diff
@@ -102,6 +103,17 @@ browser.webRequest.onBeforeRequest.addListener(
     ["blocking"]
 );
 
+// Filter out Instagram Share Identifier query parameters
+var clean_igshid = build_query_param_remover(f_match_igshid);
+browser.webRequest.onBeforeRequest.addListener(
+    clean_igshid,
+    {
+        urls: ["<all_urls>"],
```

Comment:
Is it necessarily the case that igshid appears on all urls, like fbclid does? Or just IG-internal urls?

Reply:
make sense. fixed in https://github.com/apiraino/link_cleaner/pull/13/commits/b2a8e376b0731efedf6281a9bde296a2a57264c4

Reply:
:+1: